### PR TITLE
refactor: function section has been merged with developer area

### PR DESCRIPTION
**Pull request by akhnos**

### CODE REVIEW

File: stacks/spica/packages/core/layout/home/home.layout.spec.ts [hunk 1]
```diff
@@ -17,8 +17,9 @@ import {LAYOUT_ACTIONS, LAYOUT_INITIALIZER} from "../config";
 import {ToolbarActionDirective} from "../toolbar-action";
 import {HomeLayoutComponent} from "./home.layout";
 import {CanInteractDirectiveTest} from "../../../../src/passport/directives/can-interact.directive";
+import {MatMenuModule} from "@angular/material/menu";
 
-describe("Home Layout", () => {
+fdescribe("Home Layout", () => {
```
Comment (thesayyn) [inline stacks/spica/packages/core/layout/home/home.layout.spec.ts:22]:
```suggestion
describe("Home Layout", () => {
```

File: stacks/spica/packages/core/layout/home/home.layout.ts [hunk 2]
```diff
@@ -28,14 +28,31 @@ export class HomeLayoutComponent implements OnInit {
...
+    [
+      RouteCategory.Primary,
+      {icon: "stars", index: 0, sub: {name: RouteCategory.Primary_Sub, icon: "list"}}
+    ],
+    [
+      RouteCategory.Content,
+      {
+        icon: "view_stream",
+        index: 1,
+        sub: {name: RouteCategory.Content_Sub, icon: "format_list_numbered"}
```
Comment (thesayyn) [inline stacks/spica/packages/core/layout/home/home.layout.ts:40]:
```suggestion
        children: {name: RouteCategory.Content_Sub, icon: "format_list_numbered"}
```

File: stacks/spica/packages/core/layout/home/home.layout.ts [hunk 3]
```diff
@@ -60,13 +79,16 @@ export class HomeLayoutComponent implements OnInit {
             return {
               icon: category.icon,
               category: categoryName,
-              index: category.index
+              index: category.index,
+              sub: Object.assign(
+                {items: routes.filter(route => route.category == category.sub.name)},
+                {icon: category.sub.icon}
+              )
```
Comment (thesayyn) [inline stacks/spica/packages/core/layout/home/home.layout.ts:86]:
```suggestion
              children: {
                    items: routes.filter(route => route.category == category.sub.name),
                    icon: category.sub.icon
               },
```

File: stacks/spica/packages/core/route/route.ts [hunk 4]
```diff
@@ -6,7 +6,11 @@ export enum RouteCategory {
   Content = "Buckets",
   Developer = "Developer Area",
   System = "Access Management",
-  Function = "Functions"
+
+  Primary_Sub = "Settings",
+  Content_Sub = "All Buckets",
+  Developer_Sub = "Logs",
+  System_Sub = "IAM Settings"
```
Comment (thesayyn) [inline stacks/spica/packages/core/route/route.ts:13]:
```suggestion
  System_Sub = "IA Management"
```

**Review by thesayyn — CHANGES_REQUESTED** [review]

**Comment by akhnos** [comment]
Change requests accepted

**Review by thesayyn — DISMISSED** [review]

**Review by htuna07 — APPROVED** [review]

**Pull request merged**